rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -321,7 +321,8 @@ def __next__(self):
         for i in range(len(self.output_tensor_list)):
             dimensions = self.output_tensor_list[i].dimensions()
             self.num_roi_dims = self.output_tensor_list[i].roi_dims_size()
-            self.roi_array = np.zeros(self.batch_size * self.num_roi_dims * 2, dtype=np.int32)
+            self.num_of_rois = self.num_roi_dims * 2
```

Comment:
why is this 2 times the dims?

Reply:
The number of roi dims would be n for nd array.
For each ndarray we can have x dim and y dim. hence 2 times